packages/forms/src/directives/validators.ts

```diff
@@ -140,6 +140,16 @@ abstract class AbstractValidatorDirective implements Validator {
   registerOnValidatorChange(fn: () => void): void {
     this._onChange = fn;
   }
+
+  /**
+   * @description
+   * When value of input is null or undefined return false
+   * else return true
+   */
+  enabled(): boolean {
+    return (this as unknown as {[key: string]: unknown})[this.inputName] !=
+        null /* both `null` and `undefined` */;
+  }
```

Comment:
Just for readability:
```suggestion
  enabled(): boolean {
    const inputValue = (this as unknown as {[key: string]: unknown})[this.inputName];
    return inputValue != null /* both `null` and `undefined` */;
  }
```